src/handlers/CommandHandler.ts

```diff
@@ -79,13 +79,14 @@ export class CommandHandler {
 
     add_template_hotkey(
         old_template: string | null,
-        new_template: string
+        new_template: string,
+        mode: string
     ): void {
         this.remove_template_hotkey(old_template);
 
         if (new_template) {
             this.plugin.addCommand({
-                id: new_template,
+                id: `insert-${new_template}`,
```

Comment:
Note to self: make sure changelog includes note about this breaking change. I don't think many people are using this, and I like that this is less ambiguous.